LICENSE.txt

```diff
@@ -936,6 +937,732 @@ support library is itself covered by the above license.
 
 --------------------------------------------------------------------------------
 
+3rdparty dependency gRPC is statically linked in certain binary
```

Comment:
What do you think about putting each license in its own file in a licenses directory like Apache Spark? This is easier to manage and also makes the binary licenses more visible

https://github.com/apache/spark/tree/master/licenses-binary

I think we should probably also start managing the licenses for thirdparty source inclusions in their own directory like Apache Spark.

Reply:
That is indeed nicer and more manageable! How about separating them for 1.0? I'd like to cut RC0 as soon as possible.

Reply:
Sure thing

Reply:
Created issue for it https://issues.apache.org/jira/browse/ARROW-5954